obApp.nix

```diff
@@ -19,7 +19,7 @@ in with obelisk;
     # ios.bundleIdentifier = "systems.obsidian.obelisk.examples.minimal";
     # ios.bundleName = "Obelisk Minimal Example";
 
-    __closureCompilerOptimizationLevel = "SIMPLE";
+    # __closureCompilerOptimizationLevel = "SIMPLE";
```

Comment:
@jmininger This line is unnecessary, since simple is the default.

Reply:
I think this could be removed